Updated meta descriptions flagged as 'too short' (#724)
MicrosoftDocs/quantum-docs-pr: articles/contributing/code.md

---
title: Contributing code to the Microsoft QDK

> description: Learn the guidelines for contributing sample code to the Microsoft Quantum Development Kit (QDK).

Reviewer: ```suggestion
description: Learn how to contribute sample and library code to the Microsoft Quantum Development Kit (QDK).
```
